uber/site_sections/preregistration.py

```diff
@@ -386,7 +386,7 @@ def process_group_member_payment(self, session, payment_id, stripeToken):
     def unset_group_member(self, session, id):
         attendee = session.attendee(id)
         try:
-            send_email(c.REGDESK_EMAIL, attendee.email, '{EVENT_NAME} group registration dropped',
+            send_email(c.REGDESK_EMAIL, attendee.email, '{c.EVENT_NAME} group registration dropped',
```

Comment:
This is going to raise an exception.  If you look at
https://github.com/magfest/ubersystem/blob/master/uber/utils.py#L132
you'll see that this actually needs to be ``{EVENT_NAME}`` and *not* ``{c.EVENT_NAME}``.  The way you've written it, this will raise an exception every, which means that anytime someone uses this feature we'll hit the "unable to send group unset email" block and no one will ever be notified.

I'm buried in MIVS stuff at the moment, but I'd appreciate it if someone could make a fix for this before we all forget.